docs/meshstack.azure.metering.md

```diff
@@ -5,3 +5,9 @@ title: Metering
 
 meshStack imports metering data from Azure via the Azure Cost Management API. It collects data for the previous and the current month.
 Azure only provides data for the previous day, so Azure Usage Reports shown in meshStack will not provide data for the current day.
+
+## Permission Model
```

Comment:
d: this should reference the dhall model? there should be a cross-link from the "integration" page that mentions this as one of the 3 SPNs?